docs/application/dotnet/get-started/mobile/first-app.md

```diff
@@ -102,11 +102,13 @@ To deploy and run the application on the emulator:
 
     ![Tizen Emulator Manager](media/cs_first_building_emulator_manager_mobile.png)
 
-3. Once you launch an emulator instance, you can deploy the application by clicking the emulator instance in the Visual Studio toolbar.
+3. Once you launch an emulator instance, you can deploy the application by clicking the emulator instance in the Visual Studio toolbar. Make the Mobile project **Set as StartUp Project**.
+
+    ![Deploy your package](media/vs_emulator_launch_mobile.png)
```

Comment:
I have on suggestion.
Compared to _vs_emulator_launch_tv.png_,
_vs_emulator_launch_mobile.png_ and _vs_emulator_launch_wearable.png_ have long margin at the left side.

How about cutting the left side of the images so the users can focus on the emulator instance?

Reply:
The image size of TV guides was over 900 so I corped the image. I'd like to check the layout after staging server built the PR.

When I see the staging server, the size is acceptable. I mean there is no problem to display the image having the front menu bar.